docs/reference/run.md

```diff
@@ -611,6 +612,17 @@ The following example, illustrates a dangerous way to use the flag:
 The container has unlimited memory which can cause the host to run out memory
 and require killing system processes to free memory.
 
+### Swappiness constraint
```

Comment:
@ktraghavendra thank you for the contribution.  There is some awkward phrasing so, the content is hard to follow.  How about this?

---> https://gist.github.com/moxiegirl/6a0d6e72767b1c10ad29

### Swappiness constraint

By default, a container's kernel can swap out a percentage of anonymous pages.
To set this percentage for a container, specify a `--memory-swappiness` value between 0 and 100. 
A value of 0 turns off anonymous page swapping.  A value of 100 sets all anonymous pages as swappable. For example, you can set:

    $ docker run -ti --memory-swappiness=0 ubuntu:14.04 /bin/bash

Setting the  `--memory-swappiness` option is helpful when you want to retain the container's working set and to avoid swapping performance penalties.